packages/preview/modern-unimi-thesis/0.1.0/README.md

```diff
@@ -0,0 +1,63 @@
+# modern-unimi-thesis 🎓
+
+This template is the result of the conversion from the [original LaTeX template](https://www.overleaf.com/project/641879675262cde2a670826b), in Italian.
+
+## Preview ✨
+
+<p align="center">
+  <img alt="Frontispiece/First page" src="thumbnail.png" width="45%">
+</p>
+
+> [!TIP]
+> See `docs/instructions.pdf` for more information about the template (in Italian).
```

Comment:
Can you please add a link to the file here, so that people can directly open it, without needing to open the Git repository?